packages/atomic/src/components/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -342,6 +343,12 @@ export class AtomicSearchInterface {
     window.addEventListener('hashchange', this.onHashChange);
   }
 
+  private initAriaLive() {
```

Comment:
What if we either:
- not create a default aria live component and document adding one properly in a accessibility article. Especially if we know that it doesn't work super well inside the shadow dom for some tools.
- check with a query selector if one exists before adding a second one

Reply:
> not create a default aria live component and document adding one properly in a accessibility article. Especially if we know that it doesn't work super well inside the shadow dom for some tools.

I don't mind that, but I think the current solution might deflect some maintenance cases.

> check with a query selector if one exists before adding a second one

I prefer not to do that in case an `atomic-aria-live` element is added/removed dynamically.

Reply:
> check with a query selector if one exists before adding a second one

> I prefer not to do that in case an atomic-aria-live element is added/removed dynamically.

I think that's also overkill. 

I think, either it exists at init time (and we don't expect it to be removed, because why would someone need to do that?), or it doesn't and then we need to add one.